src/arrayMethodSort.js

```diff
@@ -1,11 +1,22 @@
 'use strict';
 
-/**
- * Implement method Sort
- */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = (a, b) => {
+    return (a.toString() >= b.toString()) ? 1 : -1;
```

Comment:
Don't write it directly here. Let's create a separate default compare function

Reply:
Thank you! Wanted to avoid adding another let variable ... But it does look ugly, so I took it into consideration and implemented :)